macros/graph/plotly3D.pl

```diff
@@ -191,21 +191,18 @@ =head1 Graph3D OPTIONS
 =cut
 
 sub _plotly3D_init {
-	ADD_JS_FILE('https://cdn.plot.ly/plotly-latest.min.js', 1);
+	ADD_JS_FILE('node_modules/plotly.js-dist-min/plotly.min.js', 0, { defer => undef });
```

Comment:
Did you mean to add the `defer`ed attribute?  You didn't have that before.  That will mess things up if your javascript is not designed for that.  Although, your javascript should be designed for that if it is not.

Reply:
I forget why I didn't have it there, but following your examples it seemed to be preferred, and in my tests problem it seems to work just fine, so I added it back.

Reply:
All javascript should be loaded deferred.  Either that or loaded at the end of the page body.  No javascript should be loaded in `head` unless it is deferred or async.

Reply:
Ahh, it might be my first version of this didn't use `document.addEventListener("DOMContentLoaded", function() {` to run the javascript when the page was loaded that was why I removed it. But now all my javascript is only run on page load.